app-main/public/js/events/eventHandlers.js

```diff
@@ -23,7 +23,9 @@ export function handleClear(e) {
     tag.classList.remove("selected");
   });
 
-  submitLessonElement.textContent = "ADD LESSON";
```

Comment:
this is the statement that is making the text to ADD LESSON when we are clicking the clear button so, I think there is no use to put it here.

Reply:
Good spot, you're right 👍